sdk/monitor/opentelemetry-exporter/README.md

```diff
@@ -61,7 +61,7 @@ Coming Soon
 
 ## Examples
 
-Please take a look at the [examples](../examples) to see how to add the Azure Monitor Exporter to your existing OpenTelemetry instrumented project.
+Please take a look at the `examples` to see how to add the Azure Monitor Exporter to your existing OpenTelemetry instrumented project.
```

Comment:
I'd prefer that we remove the entire `Examples` section or replace the above line with say

`// TODO: Update this with link to samples when we have some`

Otherwise, we may miss adding the link when we have it

Reply:
Sounds good